one_liners.js

```diff
@@ -30,13 +30,13 @@ const insults = [
   'Aha, mal wieder in der Nase gebohrt, wie, %u%?',
   'Willst du mich mit deinem Geschwafel ermüden, %u%?',
   'Zu Schade, dass DAS überhaupt niemanden interessiert, %u%.',
-  'Sollt’ ich in deiner Nähe sterben,%u%, möcht’ ich, daß man mich desinfiziert!',
+  'Sollt’ ich in deiner Nähe sterben, %u%, möcht’ ich, dass man mich desinfiziert!',
   'Bist du das, %u%? Es riecht hier so nach Jauche und Dung!',
   'Ist der Blick in den Spiegel für Dich jeden Tag nicht eine Erniedrigung, %u%?',
   'Du hast soviel Sexappeal wie ein Croupier, %u%.',
   'Dein Geplänkel kommt nicht richtig in Schwung, %u%!',
   'Wurdest du damals von einem Schwein adoptiert, %u%?',
-  'Das ich nicht lache, %u%',
+  'Dass ich nicht lache, %u%',
```

Comment:
I think this one was correct before, wasn't it?

Reply:
Hmm never mind, according to https://www.korrekturen.de/forum.pl/md/read/id/10074/sbj/das-oder-dass-ich-nicht-lache/ it is "Dass".